lighting_export_json/models/product.py

```diff
@@ -536,7 +536,7 @@ def _compute_json_search_cri(self):
     @api.depends(
         'source_ids.line_ids',
         'source_ids.line_ids.cri_min',
-        'source_ids.line_ids.special_spectrum',
+        'source_ids.line_ids.special_spectrum_id',
```

Comment:
add `source_ids.line_ids.special_spectrum_id.name`, and `source_ids.line_ids.special_spectrum_id.code`,